content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: blog
+title: "Kubernetes Removals and Deprecations In 1.24"
+date: 2022-04-07
+slug: upcoming-changes-in-kubernetes-1-24
+---
+
+**Author**: Mickey Boxell (Oracle)
+
+As Kubernetes evolves, features and APIs are regularly revisited and removed. New features may offer
+an alternative or improved approach to solving existing problems, motivating the team to remove the
+old approach. Old APIs are first deprecated and then removed according to the [Kubernetes Deprecation Policy](/docs/reference/using-api/deprecation-policy/). 
+
+We want to make sure you are aware of the changes coming in the Kubernetes 1.24 release. The release will 
+**deprecate** several (beta) APIs in favor of stable versions of the same APIs. The major change coming 
+in the Kubernetes 1.24 release is the [removal of Dockershim](https://github.com/kubernetes/enhancements/issues/2221). This is discussed 
```

Comment:
I'm minded to link elsewhere. This link would be an invitation for folks who want to comment to do so on the KEP, which may not be the outcome we want.

This could happen after the article merges.